pandas/core/generic.py

```diff
@@ -165,12 +166,15 @@ def _validate_dtype(self, dtype):
 
         if dtype is not None:
             dtype = _coerce_to_dtype(dtype)
```

Comment:
Using `pandas_dtype()' here will mean that [1] will throw an error. This has been used across various tests, and all of them will break. For example, a [number](https://github.com/pandas-dev/pandas/blob/master/pandas/tests/dtypes/test_missing.py#L185) [of](https://github.com/pandas-dev/pandas/blob/master/pandas/tests/dtypes/test_missing.py#L210) [tests](https://github.com/pandas-dev/pandas/blob/master/pandas/tests/dtypes/test_missing.py#L224) [here](https://github.com/pandas-dev/pandas/blob/master/pandas/tests/dtypes/test_missing.py#L48).

[1] `pd.Series(values, dtype=object)`